packages/mui-material/src/FormControlLabel/FormControlLabel.js

```diff
@@ -158,18 +160,32 @@ const FormControlLabel = React.forwardRef(function FormControlLabel(inProps, ref
 
   const classes = useUtilityClasses(ownerState);
 
-  const typographySlotProps = slotProps.typography ?? componentsProps.typography;
+  const externalForwardedProps = {
+    slots,
+    slotProps: {
+      ...componentsProps,
+      ...slotProps,
+    },
+  };
+
+  const [TypographySlot, typographySlotProps] = useSlot('typography', {
+    elementType: Typography,
+    externalForwardedProps,
+    ownerState,
+  });
+
+  delete typographySlotProps.ownerState;
```

Comment:
Not entirely sure why deleting `ownerState` worked, but i looked previous PR's and accordion followed same pattern here https://github.com/mui/material-ui/pull/40418/files#diff-bbc688afec7bc090b1ad7bfd26cc2bedf991eca6fa3d2941b3dcbe04b3ee70a6R183

Reply:
fixed in this commit https://github.com/mui/material-ui/pull/41767/commits/60d13f67b0b63f72193489792b20c6876c8290cd

Reply:
May I ask you to split that fix into a different PR? That way, the history is cleaner, and we can revert the changes independently if necessary. Thanks!

Reply:
sure, here is the PR -> https://github.com/mui/material-ui/pull/41903. once it is merged, i'll merge next branch and ping you